deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,344 @@
+    createdAt: '2021-01-27T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v1.0.0
+    certified: 'False'
+  name: opendatahub-operator.v1.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform. Open Data Hub integrates multiple AI/ML open source components into one operator that can easily be downloaded and installed by Openshift users.
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * Spark Thrift Server - NEW A service that allows JDBC clients run Spark SQL queries.
+    * Hue - NEW A service that provides data exploration on Hive and S3 buckets. 
+    * Open Data Hub Dashboard v0.9  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * JupyterHub v3.0.7 - NEW spawner User Interface. Open source multi-user notebook platform w/ GPU support. 
+    * Elyra v0.0.5 - JupyterLab notebooks with support for AI workflows
+    * Ceph Nano v0.7 - Object Storage
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.16.0 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
+    * Airflow v1.10.11 - Workflow management
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v0.30.0 - Open source application for data exploration and visualization
+    * Apache Kafka v2.4.0 - The open source stream processing platform
+    To install one or multiple of these components use the default kfDef provided with the operator.
+
+    ### Kubeflow Components
+    ODH 1.0 supports Kubeflow v1.2.0 and some components such as KF Serving and KFP on Tekton from master branch. This will be updated when Kueflow v1.3.0 is releases. To install Kubeflow v1.2.0 components with KFP using argo please use this example KfDef (https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_openshift.v1.2.0.yaml). To install KF Serving with KFP on Tekton please use this example KfDef (https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.master.kfptekton.yaml)
+
```

Comment:
Typo "is released"

Reply:
@LaVLaS one more fix, please use the raw url for kfdef "https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.v1.2.0.yaml"